.github/workflows/pr-ui-web.yaml

```diff
@@ -0,0 +1,108 @@
+name: pr-ui-web
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "ui/web-v2/**"
+      - "!**/**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
This setting will cancel the current CI if you push something while the CI is still running.